commons/src/store/threads.ts

```diff
@@ -31,7 +36,20 @@ function initializeThreads() {
       });
       return threadsMap[queryKey];
     },
-
+    updateThread: async (
+      threadQuery: ConversationQuery,
+      queryKey: string,
+      updatedThread: Conversation,
+    ) => {
+      await updateThread(threadQuery, updatedThread).then((thread) => {
+        threadsMap[queryKey] = threadsMap[queryKey].map((_thread) => {
```

Comment:
nit - Consider renaming `_thread`, it makes the logic below a bit hard to follow. Or maybe this var can be `thread` and the current thread can be renamed to `updatedThread` to avoid confusion between the two.